runtime/parachains/src/disputes/tests.rs

```diff
@@ -907,6 +743,7 @@ fn test_provide_multi_dispute_success_and_other() {
 		let v5 = <ValidatorId as CryptoType>::Pair::generate().0;
 		let v6 = <ValidatorId as CryptoType>::Pair::generate().0;
 
+		// Mapping between key pair and `ValidatorIndex`
```

Comment:
I wonder if there is a reason for this mapping? It will be more readable if validator index matches the 'validator number' (left column).